gem_prelude.rb

```diff
@@ -17,3 +17,9 @@
 rescue LoadError
   warn "`did_you_mean' was not loaded."
 end if defined?(DidYouMean)
+
+begin
+  require 'dead_end'
```

Comment:
```suggestion
  require 'dead_end/core_ext'
```

Reply:
I'm baking this ability directly into the library with https://github.com/zombocom/dead_end/pull/142